lib/Doctrine/DBAL/Configuration.php

```diff
@@ -79,6 +80,12 @@ public function setResultCacheImpl(Cache $cacheImpl)
      */
     public function setFilterSchemaAssetsExpression($filterExpression)
     {
+        Deprecation::triggerIfCalledFromOutside(
```

Comment:
This method doesn't seem to be used internally. Tests don't count, they are not part of the library. I didn't expect this many methods having to rely on `triggerIfCalledFromOutside()`.

Reply:
Ah, so if a method is not used internally anymore then we could use the unconditional trigger indeed.

i will need to go over the methods more carefully to decide which one to use.

Reply:
@morozov i went over each call again and converted quite a few back to using `trigger`.